docs/README.md

```diff
@@ -1,39 +1,58 @@
-# Docs
+# Kyma Community Modules
 
 ## Overview
 
-The `docs` folder contains two subfolders - `user` and `contributor`. 
+The community modules are modules provided by the Kyma community and can be installed alongside the managed modules. They are not automatically updated or maintained, but they can be used to enhance your Kyma experience with additional features and capabilities.
 
-The `user` subfolder contains the end-user documentation, which is displayed on the [Kyma website](https://kyma-project.io/#/). Depending on your module needs, the subfolder must include overview, usage, or technical reference documents. To display the content on the website properly, create a `_sidebar.md` file in the `user` subfolder and list the documents it contains there. For more information on how to publish user documentation, follow [this guide](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/01-user-docs.md).
+## Quick Install
 
-The `contributor` subfolder includes any developer-related documentation to help them manually install, develop, and operate a module.
+<!-- tabs:start -->
 
-To have a common structure across all modules, all documents must be properly numbered according to the following structure:
+#### **Kyma Dashboard**
 
-> **NOTE:** It is suggested to use the following titles if you have the content that matches them; otherwise use your own, more suitable titles, or simply skip the ones you find irrelevant.
+1. Go to your Kyma dashboard and select **Modify Modules**.
 
-   - 00-xx-overview
-   - 01-xx-tutorial/configuration  
-   - 02-xx-usage  
-   - 03-xx-troubleshooting
+2. In the **Community Modules** section, select **Add** and mark the modules you want to install.
 
-where `xx` is the number of the given document. For example:
+3. Select **Add**.
+
+Your module is installed, when its status changes to `Ready`.
+
+To delete your module, select the trash icon next to the module name.
```

Comment:
Should we mark it as **Next Steps**?
We describe how to delete a community module only in Kyma dashboard. How about Kyma CLI or kubectl?